package.json

```diff
@@ -8,11 +8,13 @@
     "start:noseed": "node dist/src/index.js -- noseed",
     "dev": "npm install --frozen-lockfile --production=false && npm run build && npm run prepare-db && concurrently \"tsc --watch\" nodemon",
     "test": "cross-env NODE_ENV=test ENV=test npm run test:all",
+    "test:coverage": "cross-env NODE_ENV=test ENV=test npm run test:all:coverage",
```

Comment:
adding a script for test coverage. Still a lot of work to do:
![Uploading Screen Shot 2022-07-01 at 16.37.30.png…]()